lib/index.js

```diff
@@ -206,15 +207,11 @@ internals.test = function (title /*, options, fn */) {
     const options = arguments.length === 3 ? arguments[1] : {};
     const fn = arguments.length === 3 ? arguments[2] : arguments[1];
 
-    if (fn) {
-        Hoek.assert(fn.length === 1, 'Function for test "' + title + '" should take exactly one argument');
-    }
-
     const settings = Utils.mergeOptions(this._current.options, options, ['only']);
 
     const test = {
         path: this._path,
-        title: this._titles.join(' ') + ' ' + title,
+        title: (this._path.length ? this._titles.join(' ') + ' ' : '') + title,
```

Comment:
It’s equivalent, these arrays alway have the same length. I can change it, if you think it’s confusing.